src/LiveComponent/src/Attribute/AsLiveComponent.php

```diff
@@ -116,4 +124,13 @@ public static function liveListeners(object|string $component): array
 
         return $listeners;
     }
+
+    public static function getMethod(object|string $component): string
+    {
+        foreach ((new \ReflectionClass($component))->getAttributes(self::class) as $attr) {
+            return $attr->newInstance()->serviceConfig()['method'];
+        }
+
+        return 'post';
```

Comment:
Can be shorten like so: ?
```suggestion
       $attributes = (new \ReflectionClass($component))->getAttributes(self::class);
       
       return $attributes[0]->newInstance()->serviceConfig()['method] ?? 'post'
```

Reply:
Done